sdk/highlight-next/src/server.edge.ts

```diff
@@ -4,6 +4,7 @@ import * as withHighlightEdge from './util/with-highlight-edge'
 
 import type { HighlightEnv } from './util/types'
 export { H } from './util/highlight-edge'
+export { highlightMiddleware } from './util/highlight-middleware'
```

Comment:
I love that you noticed this. I had it as `HighlightMiddleware`, but swapped from pascal to camel case because it gets called directly and it's not wrapping anything. 

It may be utter bikeshedding, or it could be idiomatic JS. Not sure. But the camel case seems more appropriate.

![image](https://github.com/highlight/highlight/assets/878947/84a3e036-30a6-493c-9e78-0a2922275dd5)
![image](https://github.com/highlight/highlight/assets/878947/a13b1e0f-3449-4b46-b1d8-a21edf7b45b0)